scripts/zones/Xarcabard/npcs/qm4.lua

```diff
@@ -1,15 +1,16 @@
 -----------------------------------
 -- Area: Xarcabard
 -- NPC:  qm4 (???)
--- Involved in Quests: Atop the Highest Mountains
+-- Involved in Quests: Atop the Highest Mountains (for Boreal Hound)
 -- @pos -21 -25 -490 112
 -----------------------------------
 package.loaded["scripts/zones/Xarcabard/TextIDs"] = nil;
 -----------------------------------
 
-require("scripts/globals/settings");
 require("scripts/globals/keyitems");
 require("scripts/globals/quests");
+require("scripts/globals/settings");
+require("scripts/globals/status");
```

Comment:
Just curious are you trying to alphabetize the order requires land in? This is fine in this pr (don't re-change it to put it back lol) but in the future please don't go out of your way to rearrange.